README.md

```diff
@@ -556,6 +556,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Health
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Coronavirus in the UK](https://coronavirus.data.gov.uk/developers-guide) | UK Government coronavirus data, including deaths and cases by region | No | Yes | Unknown |
```

Comment:
Can I ask you to replace your link https://coronavirus.data.gov.uk/developers-guide with https://coronavirus.data.gov.uk/details/developers-guide? I can see that the link is redirected.